test/jdk/java/awt/print/Dialog/PaperSizeError.java

```diff
@@ -23,7 +23,7 @@
 
```

Comment:
could you expand wildcard imports on this test and update copyright year?

Reply:
Done.